runtime/vm/BytecodeInterpreter.hpp

```diff
@@ -5278,12 +5285,21 @@ class INTERPRETER_CLASS
 		recordJNIReturn(REGISTER_ARGS, bp);
 		restoreSpecialStackFrameLeavingArgs(REGISTER_ARGS, bp);
 
-#if JAVA_SPEC_VERSION >= 20
+#if JAVA_SPEC_VERSION >= 21
```

Comment:
`returnState` is only defined for Java 21+: please fix the condition around line 5170.

Reply:
Fixed.